diff --git a/test/helper.js b/test/helper.js
index 4f7a9a9..75933cc 100644
--- a/test/helper.js
+++ b/test/helper.js
@@ -261,7 +261,8 @@ if (!process.env.EB_NODE_COMMAND) {
                             } else if (id && result.statements && stmtFound(result.statements, id)) {
                             //if we find a block of statements and the id we are looking for, then we're good and we can continue with the testing
                                 p.resolve();
-                            } else if ((new Date(res.headers['x-experience-api-consistent-through'])).valueOf() + module.exports.getTimeMargin() >= time) {
+                            // } else if ((new Date(res.headers['x-experience-api-consistent-through'])).valueOf() + module.exports.getTimeMargin() >= time) {
+                            } else if ((new Date(res.headers['x-experience-api-consistent-through'])).valueOf() >= time) {
                             //if the desired statement has not been found, we check the con-thru header to find if the lrs is up to date and we should move on
                                 p.resolve();
                             } else {
